GG/src/Menu.cpp

```diff
@@ -465,7 +465,9 @@ void PopupMenu::Render()
             }
             std::vector<Font::LineData> lines;
             Flags<TextFormat> fmt = FORMAT_LEFT | FORMAT_TOP;
-            Pt menu_sz = m_font->DetermineLines(str, fmt, X0, lines); // get dimensions of text in menu
+            std::vector<boost::shared_ptr<Font::TextElement> > text_elements
+                = m_font->ComputeTextElements(str, fmt);
+            Pt menu_sz = m_font->DetermineLines(str, fmt, X0, text_elements, lines); // get dimensions of text in menu
```

Comment:
`X0` is magic. What does it do?  Should go into the `DetermineLines` documentation.